### ISSUE TITLE
Automate accessibility audits

### ISSUE THREAD

**Issue by kaylee42** [popcodeorg/popcode#1360]
Chrome comes with built-in accessibility auditing tools. You can try these out by clicking the audits tab in dev tools => Perform an audit => Accessibility

I believe that we can automate running this audit on popcode by using Puppeteer. There are a few considerations/steps that need to be taken for this to work: 

- confirm that Puppeteer can indeed automate this audit
- implement Puppeteer in Popcode
- configure the accessibility audit to correctly weight the different accessibility tests (ie, color contrast is very important to us, but popcode may never be fully screen reader accessible)
- create a way for snapshot queries not to be stripped out (accessibility audit is based on location, not the content on the current page)
- create a variety of snapshots testing popcode in different states
- fix current failures in accessibility audit ? 
- figure out the best way to build integration testing into the development process
